jb/itch5/generate_inside.hpp

```diff
@@ -71,12 +70,12 @@ bool record_latency_stats(
  * (before the any output is generated).
  * @returns true if the inside is affected by the change, false otherwise.
  */
-template <typename duration_t>
+template <typename duration_t, typename book_type>
```

Comment:
Ping.  We can clean this up after the fact, but we are missing a @tparam in the doxygen doc here.